src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator</RootNamespace>
-    <TargetFrameworks>net7.0;net472</TargetFrameworks>
+    <TargetFrameworks>$(NetRoslyn);net472</TargetFrameworks>
```

Comment:
This may be a place where we want a different version property here -- this is a tool that's consumed by the Code Index team, so any change to this requires a change on their side. I've verified with them they're good to take the update to 8.0 (they had already been deploying that as the newer runtime version in the last few weeks), but a change here at least means they should be notified about that.